### ISSUE TITLE
Performing segues from PageMenu built from controller within a container

### ISSUE THREAD

**Issue by britishmonkey** [uacaps/PageMenu#171]
Hi.

This is a great menu and works really well for my purposes so many thanks.

However, I need to trigger a segue from a table cell within a menu page and that is causing problems as outlined below.

**Attempted Solution 1** - directly link the segue from the table cell.
This causes a 'detached controller' error since the template for each menu page is not linked in any way to anything else in the storyboard. I assumed adding the following code would cause it to be treated as part of the storyboard but seemingly not.
`self.addChildViewController(self.pageMenu!)`
`self.view.addSubview(pageMenu!.view)`
`self.pageMenu!.didMoveToParentViewController(self)`

**Attempted Solution 2** - try to trigger the segue from the controller that built the page menu.
This resulted in 'no such segue' error since it seems to have no connection to the view that built it. No amount of 'parentController' insertion seems to work.

So now I'm a bit lost.

In short, here is my situation.

View Controller A has a Container
This container shows View Controller B
B runs the code to build PageMenu using a template view C which is a standalone, unconnected view
**Need to have a table cell within C trigger a segue to a controller D**

Many thanks!
